server/routes/testutils/appSetup.ts

```diff
@@ -91,6 +92,7 @@ function appSetup(production: boolean, testSession: Session, mockUserService: Us
   app.use('/select-location', selectLocationRoutes(standardRouter(mockUserService)))
   app.use('/incentive-summary/:locationPrefix', incentivesTableRoutes(standardRouter(mockUserService)))
   app.use('/prisoner-images/:imageId.jpeg', prisonerImagesRoutes(imageRouter()))
+  app.use('/analytics', analyticsRouter(standardRouter(mockUserService)))
```

Comment:
Sorry, this is probably needs tweaking after the middleware refactoring, it would go under [`routes/all`](https://github.com/ministryofjustice/hmpps-incentives-ui/blob/main/server/routes/all.ts).

The good news is that you don't have to repeat this in two places ☺️